### ISSUE TITLE
Accessibility: Unable to Manage Followed Blogs using Screen Reader

### ISSUE THREAD

**Issue by professarah** [Automattic/wp-calypso#16585]
#### Steps to reproduce
1. Using OSX, open Settings > Accessibility > Enable Voiceover.
<img src="https://cldup.com/Jh4zHd7Qzm-1200x1200.png"><br>
2. Navigate to  https://wordpress.com/following/manage<br>
3. Use 'tab' key to navigate between different page elements.<br>
4. Land on the list of blogs.<br>

#### What I expected
To be able to select and interact with different blogs and the options within the table itself.

#### What happened instead
It only gave the option to select the table, stating: "You are currently on a table."<br>
<img src="https://cldup.com/YSR5fD9Qt8-1200x1200.png">
No elements can be selected inside this table.

We can select and manipulate other elements on a page, but not this one. 

#### Browser / OS version
Not browser/OS specific

#### Context / Source
Follow-Up: 3339517-t
First Reported: 531284-hc
#user-report

This is a #painpoint for our users with visual impairments, as this makes it extremely difficult to manage their own blog subscriptions with assistance.

**Comment by bluefuton** [comment]
Thanks @swoolz! I've confirmed the issue and it's not possible to enter the grid containing the user's subscriptions.

The first step to getting this solved is:

https://github.com/bvaughn/react-virtualized/pull/744

It's just been merged into `react-virtualized` and we need to wait for the next release (hopefully in the next week or so) and then update Calypso to the new version. Afterwards, we need to add `role="row"` on our `reader-infinite-stream__row-wrapper`. The grid should then be available to screen readers 👍

**Comment by bluefuton** [comment]
A user of react-virtualized suggests that `role="tabpanel"` on the list may be worth a try:

https://github.com/bvaughn/react-virtualized/issues/819#issuecomment-342548476

**Comment by senff** [comment]
Customer asked for a status. I promised I would ask (or at least make a note) and let them know. 
1850549-HC

**Comment by omidfi** [comment]
I tested this on Firefox and Chrome and seems like it's resolved, I can tab through the list of blogs: 

![calypso-manage](https://user-images.githubusercontent.com/7697632/37324975-84b9e1fe-2694-11e8-96e0-5573d83e13c8.gif)

**Comment by xpurichan** [comment]
User reports that this is an ongoing issue for them.

They only use Safari since only integrates with Apple VoiceOver. They say that this is not only an issue on the "Followed Sites" (https://wordpress.com/following/manage) page, but also the "Followers" page (https://wordpress.com/people/followers/travelswithtio.com).

We have an open ticket with them, in case we need to ask them questions or when we're able to provide them with an update.

Ticket: Zen-1083473

**Comment by rw-ye** [comment]
5715452-zd-woothemes - The user reached out again about this today. They are still experiencing issues with using Apple VoiceOver to navigate their followed blogs.

**Comment by roo2** [comment]
ok, it looks like the react-vertualized component has been updated and we can now pass `aria-hidden=false` to the component. This allows me to tab through the table where I couldn't before, it will also allow the screenreader to access this table now. I couldn't reproduce the issue on the followers list page or see aria-hidden in the markup, I wonder if that has been fixed there already? 

PR https://github.com/Automattic/wp-calypso/pull/70090
